frontend/src/metabase/query_builder/actions.js

```diff
@@ -338,8 +339,13 @@ export const initializeQB = (location, params) => {
     if (params.cardId || serializedCard) {
       // existing card being loaded
       try {
-        // if we have a serialized card then unpack it and use it
-        card = serializedCard ? deserializeCardFromUrl(serializedCard) : {};
+        // if we have a serialized card then unpack it, normalize syntax and use it
+        if (serializedCard) {
+          card = deserializeCardFromUrl(serializedCard)
+          card.dataset_query = normalize(card.dataset_query)
```

Comment:
@camsaul can you please help me with `normalize` function returning not exactly correct query?
What expected is on the left side, the actual is on the right
<img width="978" alt="Screen Shot 2021-04-06 at 15 28 05" src="https://user-images.githubusercontent.com/14301985/113710684-b18a2100-96ec-11eb-8e90-21b99c45f7fa.png">